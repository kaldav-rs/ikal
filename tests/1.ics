BEGIN:VCALENDAR
PRODID:-//Nextcloud calendar v1.5.0
VERSION:2.0
CALSCALE:GREGORIAN
BEGIN:VEVENT
CREATED:20170209T192358
DTSTAMP:20170209T192358
LAST-MODIFIED:20170209T192358
UID:5UILHLI7RI6K2IDRAQX7O
SUMMARY:Vers
CLASS:PUBLIC
STATUS:CONFIRMED
DTSTART;VALUE=DATE-TIME;TZID=Europe/Paris:20150219T190000
DTEND;VALUE=DATE:20170210
END:VEVENT
END:VCALENDAR
