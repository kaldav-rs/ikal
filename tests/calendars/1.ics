BEGIN:VCALENDAR
PRODID:-//Nextcloud calendar v1.5.0
VERSION:2.0
CALSCALE:GREGORIAN
BEGIN:VEVENT
CREATED:20170209T192358
DTSTAMP:20170209T192358
LAST-MODIFIED:20170209T192358
UID:5UILHLI7RI6K2IDRAQX7O
SUMMARY:Vers
CLASS:PUBLIC
STATUS:CONFIRMED
DTSTART;VALUE=DATE-TIME;TZID=Europe/Paris:20150219T190000
DTEND;VALUE=DATE:20170210
END:VEVENT
BEGIN:VTIMEZONE
TZID:Europe/Berlin
X-LIC-LOCATION:Europe/Berlin
BEGIN:DAYLIGHT
TZOFFSETFROM:+0100
TZOFFSETTO:+0200
TZNAME:CEST
DTSTART:19700329T020000
RRULE:FREQ=YEARLY;BYDAY=-1SU;BYMONTH=3
END:DAYLIGHT
BEGIN:STANDARD
TZOFFSETFROM:+0200
TZOFFSETTO:+0100
TZNAME:CET
DTSTART:19701025T030000
RRULE:FREQ=YEARLY;BYDAY=-1SU;BYMONTH=10
END:STANDARD
END:VTIMEZONE
END:VCALENDAR
