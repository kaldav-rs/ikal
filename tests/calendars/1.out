Ok(
    VCalendar {
        prodid: "-//Nextcloud calendar v1.5.0",
        version: "2.0",
        calscale: Some(
            "GREGORIAN",
        ),
        method: None,
        alarms: [],
        events: [
            VEvent {
                dtstamp: 2017-02-09T19:23:58+01:00,
                uid: "5UILHLI7RI6K2IDRAQX7O",
                dtstart: 2015-02-19T19:00:00+01:00,
                class: Some(
                    Public,
                ),
                created: Some(
                    2017-02-09T19:23:58+01:00,
                ),
                description: None,
                geo: None,
                last_modified: Some(
                    2017-02-09T19:23:58+01:00,
                ),
                location: None,
                organizer: None,
                priority: None,
                sequence: None,
                status: Some(
                    Confirmed,
                ),
                summary: Some(
                    "Vers",
                ),
                transp: None,
                url: None,
                recurid: None,
                rrule: None,
                dtend: Some(
                    2017-02-10T00:00:00+01:00,
                ),
                duration: None,
                attach: [],
                attendee: [],
                categories: [],
                comment: [],
                contact: [],
                exdate: [],
                rstatus: [],
                related_to: [],
                resources: [],
                rdate: [],
                x_prop: {},
                iana_prop: {},
            },
        ],
        freebusy: [],
        journals: [],
        todo: [],
        timezones: [
            VTimezone {
                tzid: "Europe/Berlin",
                last_modified: None,
                tzurl: None,
                standard: [
                    Prop {
                        dtstart: 1970-10-25T03:00:00+01:00,
                        tzoffsetto: +01:00,
                        tzoffsetfrom: +02:00,
                        rrule: Some(
                            Recur {
                                freq: Yearly,
                                until: None,
                                count: None,
                                interval: None,
                                by_second: [],
                                by_minute: [],
                                by_hour: [],
                                by_day: [],
                                by_monthday: [],
                                by_yearday: [],
                                by_weekno: [],
                                by_month: [
                                    10,
                                ],
                                by_setpos: [],
                                wkst: None,
                            },
                        ),
                        comment: [],
                        rdate: [],
                        tzname: [
                            "CET",
                        ],
                        x_prop: {},
                        iana_prop: {},
                    },
                ],
                daylight: [
                    Prop {
                        dtstart: 1970-03-29T02:00:00+01:00,
                        tzoffsetto: +02:00,
                        tzoffsetfrom: +01:00,
                        rrule: Some(
                            Recur {
                                freq: Yearly,
                                until: None,
                                count: None,
                                interval: None,
                                by_second: [],
                                by_minute: [],
                                by_hour: [],
                                by_day: [],
                                by_monthday: [],
                                by_yearday: [],
                                by_weekno: [],
                                by_month: [
                                    3,
                                ],
                                by_setpos: [],
                                wkst: None,
                            },
                        ),
                        comment: [],
                        rdate: [],
                        tzname: [
                            "CEST",
                        ],
                        x_prop: {},
                        iana_prop: {},
                    },
                ],
                x_prop: {
                    "X-LIC-LOCATION": "Europe/Berlin",
                },
                iana_prop: {},
            },
        ],
        x_prop: {},
        iana_prop: {},
    },
)
